### ISSUE TITLE
Support for Laravel 5.2

### ISSUE THREAD

**Issue by JoseClaudioADS** [jarektkaczyk/eloquence#42]
When I upgrade to Laravel 5.2. I noticed a compile error in Builder.php File Sofa \ Hookable package. Stating that the implementation of the method "pluck" should be consistent with the implementation of the file "Illuminate \ Database \ Eloquent \ Builder". 

I realized that at that lacked a parameter ($ key) in the method signature.

I did the manual implementation. But I wonder when we update the version of eloquence with all that it takes.

**Comment by RomainLanz** [comment]
Same as here https://github.com/jarektkaczyk/hookable/issues/4

**Comment by DevNIX** [comment]
Why don't you make a pull request? I have that problem too :D

**Comment by DevNIX** [comment]
Lol nevermind, I've just read this https://github.com/jarektkaczyk/hookable/pull/3

**Comment by cornex** [comment]
Everything i did was to navigate to ```vendor/sofa/hookable/Builder.php``` and changed:
```php
public function pluck($column)
```
to
```php
public function pluck($column, $key = NULL)
```
Works for me haha

**Comment by mydnic** [comment]
@cornex :+1: 
Any ETA on the package update ?

**Comment by cornex** [comment]
@mydnic , i pushed the changes to a own repo until they have updated it. Feel free to use it if you need https://github.com/cornex/eloquence

**Comment by DevNIX** [comment]
@jarektkaczyk said he will fix it ASAP on this issue, I would wait for his bugfix  (https://github.com/jarektkaczyk/hookable/pull/3)

**Comment by cornex** [comment]
@DevNIX , yeah I just saw that one. Thanks! :+1:

**Comment by jarektkaczyk** [comment]
Thanks for your patience folks!

The fix is out, latest v5.2.1 works fine (and it passes all the tests, except for PHP7 which complains about Mockery - will fix that dep later).
